2021-05-08 12:12:19,375 INFO    MainThread:26588 [wandb_setup.py:_flush():69] setting env: {}
2021-05-08 12:12:19,377 INFO    MainThread:26588 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-08 12:12:19,377 INFO    MainThread:26588 [wandb_init.py:_log_setup():336] Logging user logs to /n/holyscratch01/Academic-cluster/Spring_2021/g_84102/SCRATCH/ImageNet/wandb/run-20210508_121219-33irnup2/logs/debug.log
2021-05-08 12:12:19,378 INFO    MainThread:26588 [wandb_init.py:_log_setup():337] Logging internal logs to /n/holyscratch01/Academic-cluster/Spring_2021/g_84102/SCRATCH/ImageNet/wandb/run-20210508_121219-33irnup2/logs/debug-internal.log
2021-05-08 12:12:19,378 INFO    MainThread:26588 [wandb_init.py:init():369] calling init triggers
2021-05-08 12:12:19,378 INFO    MainThread:26588 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {}
2021-05-08 12:12:19,378 INFO    MainThread:26588 [wandb_init.py:init():418] starting backend
2021-05-08 12:12:19,378 INFO    MainThread:26588 [backend.py:_multiprocessing_setup():69] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-05-08 12:12:19,446 INFO    MainThread:26588 [backend.py:ensure_launched():123] starting backend process...
2021-05-08 12:12:19,484 INFO    MainThread:26588 [backend.py:ensure_launched():127] started backend process with pid: 26650
2021-05-08 12:12:19,487 INFO    MainThread:26588 [wandb_init.py:init():423] backend started and connected
2021-05-08 12:12:19,490 INFO    MainThread:26588 [wandb_init.py:init():465] updated telemetry
2021-05-08 12:12:19,491 INFO    MainThread:26588 [wandb_init.py:init():484] communicating current version
2021-05-08 12:12:21,639 INFO    MainThread:26588 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.10.30 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-05-08 12:12:21,640 INFO    MainThread:26588 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-05-08 12:12:21,803 INFO    MainThread:26588 [wandb_init.py:init():522] starting run threads in backend
2021-05-08 12:12:25,299 INFO    MainThread:26588 [wandb_run.py:_console_start():1524] atexit reg
2021-05-08 12:12:25,302 INFO    MainThread:26588 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2021-05-08 12:12:25,303 INFO    MainThread:26588 [wandb_run.py:_redirect():1376] Redirecting console.
2021-05-08 12:12:25,305 INFO    MainThread:26588 [wandb_run.py:_redirect():1438] Redirects installed.
2021-05-08 12:12:25,305 INFO    MainThread:26588 [wandb_init.py:init():546] run started, returning control to user process
2021-05-08 12:12:25,306 INFO    MainThread:26588 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 1
2021-05-08 12:12:25,307 INFO    MainThread:26588 [wandb_run.py:_restore():1466] restore
2021-05-08 12:12:27,458 INFO    MainThread:26588 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 567
  total_bytes: 567
}

2021-05-08 12:12:27,857 INFO    MainThread:26588 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 567
  total_bytes: 6167
}

2021-05-08 12:12:28,040 INFO    MainThread:26588 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 857
  total_bytes: 6167
}

2021-05-08 12:12:28,143 INFO    MainThread:26588 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6119
  total_bytes: 6167
}

2021-05-08 12:12:28,245 INFO    MainThread:26588 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6167
  total_bytes: 6167
}

2021-05-08 12:12:28,348 INFO    MainThread:26588 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6167
  total_bytes: 6167
}

2021-05-08 12:12:28,451 INFO    MainThread:26588 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6167
  total_bytes: 6167
}

2021-05-08 12:12:28,553 INFO    MainThread:26588 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6167
  total_bytes: 6167
}

2021-05-08 12:12:28,656 INFO    MainThread:26588 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6167
  total_bytes: 6167
}

2021-05-08 12:12:30,059 INFO    MainThread:26588 [wandb_run.py:_show_files():1838] logging synced files
